class.jetpack.php

```diff
@@ -633,6 +638,28 @@ private function __construct() {
 		add_filter( 'jetpack_sync_before_send_updated_option', array( $this, 'jetpack_track_last_sync_callback' ), 99 );
 	}
 
+	function point_edit_links_to_calypso( $default_url, $post_id ) {
+		if ( ! $post = get_post( $post_id ) ) {
+			return $default_url;
+		}
+
+		$post_type = $post->post_type;
+
+		if ( in_array( $post_type, array( 'post', 'page' ) ) ) {
+			$path_prefix = $post_type;
+		} else if ( in_array( $post_type, apply_filters( 'rest_api_allowed_post_types', array( 'post', 'page', 'revision' ) ) ) ) {
+			$path_prefix = sprintf( 'edit/%s', $post_type );
+		}
+
+		if ( ! isset( $path_prefix ) ) {
+			return $default_url;
+		}
+
+		$site_slug  = Jetpack::build_raw_urls( get_home_url() );
+
+		return esc_url_raw( sprintf( 'https://wordpress.com/%s/%s/%d', $path_prefix, $site_slug, $post_id ) );
```

Comment:
Is there a general method we can use to get the base portion of this url (`'https://wordpress.com'`)? If there is, should we use it?

Reply:
Not that I know of, but maybe someone from Jetpack crew can correct me. I don't expect this URL to change anytime soon - any other reason to prefer the general method there?

Reply:
> I don't expect this URL to change anytime soon
👍 

> any other reason to prefer the general method there?

Nope. No other reason here.